content/blog/2025/07/2025-07-11-twim.md

```diff
@@ -0,0 +1,199 @@
+
+[taxonomies]
+author = ["HarHarLinks"]
+category = ["This Week in Matrix"]
++++
+
+## Matrix Live
+
+{{youtube_player(video_id="xaRCGlkMcf0")}}
+  
+</br>
+
+You can find the Matrix Foundation's Events Working Group's room at [#events-wg:matrix.org](https://matrix.to/#/%23events-wg%3Amatrix.org), all about Matrix Community Summit [on their website](https://matrix-community.events/conferences/2025-summit.html), and Matrix Community Retreats at [#workation-nation-general](https://matrix.to/#/#workation-nation-general:datanauten.de).
+
+## Dept of Spec 📜
+
+[Andrew Morgan (anoa) {he/him}](https://matrix.to/#/@andrewm:element.io) says
+
+> Here's your weekly spec update! The heart of Matrix is the specification - and this is modified by Matrix Spec Change (MSC) proposals. Learn more about how the process works at <https://spec.matrix.org/proposals>.
+> 
+> ### MSC Status
+> 
+> **New MSCs:**
+> 
+> * [MSC4310: MatrixRTC decline `m.rtc.notify`](https://github.com/matrix-org/matrix-spec-proposals/pull/4310)
+> * [MSC4309: Finalised delayed event on sync](https://github.com/matrix-org/matrix-spec-proposals/pull/4309)
+> * [MSC4308: Thread Subscriptions extension to Sliding Sync](https://github.com/matrix-org/matrix-spec-proposals/pull/4308)
+> * [MSC4307: Validate that `auth_events` are in the correct room](https://github.com/matrix-org/matrix-spec-proposals/pull/4307)
+> * [MSC4306: Thread Subscriptions](https://github.com/matrix-org/matrix-spec-proposals/pull/4306)
+> * [MSC4291: Room IDs as hashes of the create event](https://github.com/matrix-org/matrix-spec-proposals/pull/4291)
+> 
+> **MSCs in Final Comment Period:**
+> 
+> * [MSC4222: Adding `state_after` to `/sync`](https://github.com/matrix-org/matrix-spec-proposals/pull/4222) (merge)
+> * [MSC4210: Remove legacy mentions](https://github.com/matrix-org/matrix-spec-proposals/pull/4210) (merge)
+> 
+> **Accepted MSCs:**
+> 
+> * _No MSCs were accepted this week._
+> 
+> **Closed MSCs:**
+> 
+> * [MSC1998: Two-Factor Authentication Providers](https://github.com/matrix-org/matrix-spec-proposals/pull/1998)
+> * [MSC3741: Revealing the useful login flows to clients after a soft logout](https://github.com/matrix-org/matrix-spec-proposals/pull/3741)
+> * [MSC3105: Previewing UIA flows](https://github.com/matrix-org/matrix-spec-proposals/pull/3105)
+>     - The above 3 MSCs were closed due to dealing with the old auth system, which has been made obsolete by Matrix's next-gen auth system.
+> * [\[WIP\] MSC4051: Using the create event as the room ID](https://github.com/matrix-org/matrix-spec-proposals/pull/4051)
+>     - Superseded by [MSC4291](https://github.com/matrix-org/matrix-doc/pull/4291)
+
+[tulir](https://matrix.to/#/@tulir:maunium.net) announces
+
+> #### MSC highlight: Room IDs as hashes of the create event
+> You may have noticed some placeholder MSCs being opened in the past half a year. One of them has now received content ahead of the others:
```

Comment:
```suggestion
 ### MSC highlight: Room IDs as hashes of the create event

[tulir](https://matrix.to/#/@tulir:maunium.net) announces

> You may have noticed some placeholder MSCs being opened in the past half a year. One of them has now received content ahead of the others:
```

All others seem to be flipped like this so we might as well. At least the header level.